resources/views/sprint/view.blade.php

```diff
@@ -163,4 +163,19 @@
 
 @section('optional_scripts')
 	{!! HTML::script('js/sprint_overview.js') !!}
+	{!! HTML::script('js/datepicker.js') !!}
+
+	<script type="text/javascript">
+		var settings = {
+			format: 'yyyy-mm-dd',
+			autoclose: true
+		};
+
+		$('.datepicker.start').datepicker(settings);
+		$('.datepicker.end').datepicker(settings);
+	</script>
+@stop
+
+@section('optional_styles')
+	{!! HTML::style('/css/datepicker.css') !!}
```

Comment:
It's a bit ugly to add those scripts and styles here but somehow they don't work if I put them into settings_form.blade.php. @jakob-WMDE thoughts?